docs/api-markdown-documenter/index.js

```diff
@@ -4,16 +4,41 @@
  */
 
 const chalk = require("chalk");
-
+const versions = require("../data/versions.json");
 const { renderApiDocumentation } = require("./render-api-documentation");
 
-renderApiDocumentation().then(
-	() => {
-		console.log(chalk.green("API docs written!"));
-		process.exit(0);
-	},
-	(error) => {
-		console.error("API docs could not be written due to an error:", error);
-		process.exit(1);
-	},
-);
+/**
+ * This is the enable/disable flag for rendering multiple versions of the API docs. (now: false)
```

Comment:
This comment is attached to the `renderApiAllDocumentation` function, but it seems to be describing the argument.

I think we should re-work this to document what the whole script does, and what its expected parameters are. That way anyone who wants to know how to invoke it via something like `node api-markdown-documenter true` knows what it does and how to call it.